src/analysis/calculus/times_cont_diff.lean

```diff
@@ -2121,10 +2101,71 @@ lemma times_cont_diff.sum
   {ι : Type*} {f : ι → E → F} {s : finset ι} {n : with_top ℕ}
   (h : ∀ i ∈ s, times_cont_diff 𝕜 n (λ x, f i x)) :
   times_cont_diff 𝕜 n (λ x, (∑ i in s, f i x)) :=
-begin
-  simp [← times_cont_diff_on_univ] at *,
-  exact times_cont_diff_on.sum h
-end
+by simp [← times_cont_diff_on_univ] at *; exact times_cont_diff_on.sum h
+
+/-! ### Product of two functions -/
+
+/- The product is smooth. -/
+lemma times_cont_diff_mul {n : with_top ℕ} :
+  times_cont_diff 𝕜 n (λ p : 𝕜 × 𝕜, p.1 * p.2) :=
```

Comment:
Would it make sense to do this lemma for `R × R`, where `R` is any normed algebra over `𝕜`, instead of for `𝕜 × 𝕜`?  You could still do the `𝕜 × 𝕜` version as a corollary.  But if you do the normed algebra version, you have nearly finished showing that `units R` is a Lie group.

Reply:
Yeah sure I did not think about this application (I had other purposes in mind), but it sounds like a great idea I'll add it!

Reply:
I still need to do this, I'll do it tomorrow I won't forget

Reply:
@Nicknamen Just bumping this change request, I know it was buried in the 100 others above :)